e2e/runner/sample-apps-shared/helpers/prepare-app.ts

```diff
@@ -29,17 +30,28 @@ export function copyExampleEnvFile({
 }
 
 export function copyLocalMetabaseJar(rootPath: string) {
-  const metabaseJarToPath = path.join(rootPath, LOCAL_DIST_PATH);
+  const destinationPath = path.join(rootPath, LOCAL_DIST_PATH);
 
-  fs.cpSync(METABASE_JAR_DIST_PATH, metabaseJarToPath, { recursive: true });
+  fs.cpSync(METABASE_JAR_DIST_PATH, destinationPath, { recursive: true });
 }
 
 export function copyLocalEmbeddingSdkPackage(rootPath: string) {
-  const embeddingSdkToPath = path.join(
+  const destinationPath = path.join(rootPath, LOCAL_DIST_PATH, "embedding-sdk");
+
+  fs.cpSync(EMBEDDING_SDK_DIST_PATH, destinationPath, { recursive: true });
+}
+
+export function copyShoppyMetabaseAppDBDump(rootPath: string) {
+  const sourcePath = path.join(
+    E2E_TMP_FOLDER_PATH,
+    "db_dumps/shoppy_metabase_app_db_dump.sql",
+  );
+
+  const destinationPath = path.join(
     rootPath,
     LOCAL_DIST_PATH,
-    "embedding-sdk",
+    "metabase_dump.sql",
   );
 
-  fs.cpSync(EMBEDDING_SDK_DIST_PATH, embeddingSdkToPath, { recursive: true });
+  fs.copyFileSync(sourcePath, destinationPath);
```

Comment:
What is the difference between `copyFileSync` and `cpSync`? It feels we should use the same command if we could.